curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5efb2c990dc218d6c85f89b2.md

```diff
@@ -9,7 +9,7 @@ dashedName: step-42
 
 Even though you added your button below the text input, they appear next to each other on the page. That's because both `input` and `button` elements are <dfn>inline elements</dfn>, which don't appear on new lines.
 
-You learned previously that the button submits the form by default, but you can explicitly add the `type` attribute with the value `submit` to make it clearer. Go ahead and do this to specify where this button should submit the form.
+The button you added will submit the form by default. But relying on default behavior may cause confusion. Add the `type` attribute with the value `submit` to the `button` to make it clear that it is a submit button.
```

Comment:
Starting a sentence with a conjunction is technically grammatically correct, but it's an uncommon enough construction that I think it might cause a bit of confusion. What about this version?

```suggestion
The button you added will submit the form by default. However, relying on default behavior may cause confusion. Add the `type` attribute with the value `submit` to the `button` to make it clear that it is a submit button.
```